### ISSUE TITLE
Spam in source files

### ISSUE THREAD

**Issue by NexAdn** [TheWiseNoob/OMP#14]
I just checked Playback.h and had to look in line 73 to find the first real code (if you would call preprocessor directives code) and line 200 (!) to find the actual class that's being declared by Playback.h. The header of this file (and I suppose the rest of the files, too) is just full of spam. License information could be placed at the bottom of the source code where it bothers nobody. Also there is no need to list the dependencies in every file. Additional information that has nothing to do with the code itself belongs to the documentation or the README.md. 

Also these much spaces and comments make the code much less readable to me and (as I suppose) to others. Having some comments briefly explain some methods of the class is okay, but e.g. destructors don't need any comment as their purpose is clear by definition as well as their arguments. Getters often times don't need any description either as their name is most times self-describing.

In fact, your coding and commenting style is way too verbose. By removing all unneccesary empty lines (keeping an empty line between different sections as one normally does with his code) I could reduce your Playback.h from 1005 (!) lines (which is way too much lines for this little class) to 170 lines (or 210 including your copyright notice and a newline before EOF).

TL;DR you should really consider changing your coding style as the current one makes it very hard to read your code and contribute PRs.

**Comment by TheWiseNoob** [comment]
Not interested in your criticism of my coding style. Please don't open another issue like this and waste my time.
